bin/daenerys

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env php
+<?php
+function includeIfExists($file) 
+{
+    if (file_exists($file)) {
+        return include $file;
+    }
+}
+
+// Dance to find the autoloader.
+// TOOD: change this to open up the Composer config and use $c['config']['vendor-dir'] instead of "vendor"
+includeIfExists(getcwd() . '/vendor/autoload.php') ||
+includeIfExists(__DIR__ . '/../vendor/autoload.php') ||
+includeIfExists(__DIR__ . '/../autoload.php');
```

Comment:
Why using such a strange construct? __DIR__ should palways oint to the dir the file is in, even if called from directories above. All you need is the second line, right?